app/models/financialDetails/WhatYouOweChargesList.scala

```diff
@@ -49,12 +50,17 @@ case class WhatYouOweChargesList(balanceDetails: BalanceDetails, chargesList: Li
       && overdueChargeList.exists(_.lpiWithDunningLock.getOrElse[BigDecimal](0) > 0)) true
     else false
 
-
   def interestOnOverdueCharges: Boolean =
     if (overdueChargeList.exists(_.interestOutstandingAmount.isDefined)
       && overdueChargeList.exists(_.latePaymentInterestAmount.getOrElse[BigDecimal](0) <= 0)) true
     else false
 
+  def getRelevantDueDate: LocalDate =
+    outstandingChargesModel.get.bcdChargeType.get.relevantDueDate.getOrElse(throw MissingFieldException("documentRelevantDueDate"))
+
+  def getAciChargeWithTieBreakerChargeAmount: BigDecimal =
+    outstandingChargesModel.get.getAciChargeWithTieBreaker.getOrElse(throw MissingFieldException("documentAciChargeWithTieBreaker")).chargeAmount
```

Comment:
similar situation here, we need to handle situation where there is no value in `getAciChargeWithTieBreaker`